sound/soc/intel/boards/sof_cs42l42.c

```diff
@@ -493,7 +493,7 @@ MODULE_DEVICE_TABLE(platform, board_ids);
 static struct platform_driver sof_audio = {
 	.probe = sof_audio_probe,
 	.driver = {
-		.name = "sof_cs42l42",
+		.name = "cs42l42",
```

Comment:
that doesn't seem right to me, are you confusing with the card name

````
static struct snd_soc_card sof_audio_card_cs42l42 = {
	.name = "cs42l42", /* the sof- prefix is added by the core */
````

Reply:
ah yes i am, thanks